includes/Endpoints/Status.php

```diff
@@ -87,5 +93,29 @@ public function get_status(WP_REST_Request $request) {
         // Use the Response helper for a standard format
         return Response::success($success_message, $data);
     }
-    
+
+    /**
+     * Fetches the latest version of the plugin from the GitHub repository.
+     *
+     * @return string|null The latest version tag or null if unable to retrieve.
+     */
+    protected function fetch_latest_version_from_github() {
+        // GitHub URL for the specific repository
+        $url = "https://api.github.com/repos/vplugins/sm-post-connector/releases/latest";
```

Comment:
Add this path in `includes/Helper/Globals.php` and render it from there.

Reply:
Sure @rajanvijayan

Reply:
@rajanvijayan Please review the changes; the path was already present in Globals.php and has now been rendered.